apps/planning-suite-kit/package.json

```diff
@@ -6,7 +6,7 @@
     "coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:test",
     "ganache-cli:test": "../shared/test-helpers/ganache-cli.sh",
     "lint": "solium --dir ./contracts",
-    "start:kit": "aragon run --template PlanningKit --template-init @ARAGON_ENS --client-version e08deb62080e17fe652fa83f60dc81a02c455c41",
+    "start:kit": "../../node_modules/@aragon/cli-monorepo/packages/aragon-cli/dist/cli.js run --template PlanningKit --template-init @ARAGON_ENS --client-version 5199d2f3356566a5bf5a5dc74ebc61a7d9b3d32a --client-repo https://github.com/AutarkLabs/aragon",
```

Comment:
This is only working on your computer, please revert the line to use the global `aragon` package command instead of your locally downloaded `@aragon/cli`

The newest @aragon/cli has some compatibility issues and this change involves some sensible issues with the templates and some contracts, we should not just try to change this without proper testing.

Reply:
We shouldn't try to merge non-working code

Reply:
This PR is still marked as draft. I had no intention of merging it with this line as it is 😄